projects/schematics/src/shared/lib-configs/product-multi-dimensional-schematics-config.ts

```diff
@@ -41,20 +36,11 @@ export const PRODUCT_MULTI_DIMENSIONAL_LIST_SCHEMATICS_CONFIG: SchematicConfig =
     },
     folderName: PRODUCT_MULTI_DIMENSIONAL_FOLDER_NAME,
     moduleName: PRODUCT_MULTI_DIMENSIONAL_LIST_MODULE_NAME,
-    featureModule: [
-      {
-        name: PRODUCT_MULTI_DIMENSIONAL_LIST_MODULE,
-        importPath: SPARTACUS_PRODUCT_MULTI_DIMENSIONAL_LIST,
-      },
-    ],
+    featureModule: [],
```

Comment:
`medium:` Let's remove this property if empty.

Reply:
It's required, I can't remove it.
```typescript
export interface SchematicConfig {
...
  featureModule: Module | Module[];
...
}
```